pkg/test/framework/components/istio/operator.go

```diff
@@ -61,7 +61,7 @@ const (
 
 var (
 	// the retry options for waiting for an individual component to be ready
-	componentDeployTimeout = retry.Timeout(30 * time.Second)
+	componentDeployTimeout = retry.Timeout(1 * time.Minute)
```

Comment:
I wonder if you needed to increase this because you had previously removed the concurrency when installing? Can you verify that you still need this change?